### ISSUE TITLE
exit code crash

### ISSUE THREAD

**Issue by kissgyorgy** [kissgyorgy/cloudflare-dyndns#16]
with v4.0-beta2:
```
Traceback (most recent call last):
  File "/app/.local/bin/cloudflare-dyndns", line 5, in <module>
    main()
  File "/app/.local/lib/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/app/.local/lib/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/app/.local/lib/python3.9/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/app/.local/lib/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/app/.local/lib/python3.9/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/app/cloudflare_dyndns/cli.py", line 246, in main
    exit_codes.remove(0)
KeyError: 0
```

**Comment by deaglenomics** [comment]
Not sure if it helps but I was getting that error but it went away after I deleted the original API token and  recreated it.

**Comment by kissgyorgy** [comment]
This has been fixed in v4.0-beta4!
